src/scales/varia.h

```diff
@@ -56,7 +56,8 @@ class VariaScalesPlugin {
     return !deviceName.empty() &&
       (
         deviceName.find("AKU MINI SCALE") == 0 ||
-        deviceName.find("VARIA AKU") == 0
+        deviceName.find("VARIA AKU") == 0 ||
```

Comment:
I think this line can be removed now.

Reply:
Happy to if we're 100% certain that variant isn't in use

Reply:
Hmm true, harmless to leave in. Never know what older firmware might have.